### PR TITLE
Add manual coverage checking for parse_content_range()

### PR DESCRIPTION
**Pull request by jberglinds**

### CODE REVIEW

**Review by robban112 — APPROVED** [review]

File: httpie/downloads.py [hunk 1]
```diff
@@ -71,6 +75,10 @@ def parse_content_range(content_range, resumed_from):
         if content_range_dict['instance_length']
```
Comment (nedsko) [inline httpie/downloads.py]: Does this need to be considered a branch as well?

File: httpie/downloads.py [hunk 2]
```diff
@@ -81,12 +89,14 @@ def parse_content_range(content_range, resumed_from):
     if (first_byte_pos >= last_byte_pos or
             (instance_length is not None and
              instance_length <= last_byte_pos)):
+        has_branched('parse_content_range', 5)
         raise ContentRangeError(
             'Invalid Content-Range returned: %r' % content_range)
 
     if (first_byte_pos != resumed_from or
             (instance_length is not None and
```
Comment (nedsko) [inline httpie/downloads.py]: Separate to multiple if:s?

File: httpie/downloads.py [hunk 3]
```diff
@@ -81,12 +89,14 @@ def parse_content_range(content_range, resumed_from):
     if (first_byte_pos >= last_byte_pos or
             (instance_length is not None and
```
Comment (nedsko) [inline httpie/downloads.py]: Separate to multiple if:s?

**Comment by ghost** [comment]
Yeah, I think all if-statements should be considered as branches. Additionally, or & and statements needs to be split up. I can patch this.

File: httpie/downloads.py [hunk 4]
```diff
@@ -60,39 +63,61 @@ def parse_content_range(content_range, resumed_from):
...
+        has_branched('parse_content_range', 3)
+        instance_length = int(content_range_dict['instance_length'])
+    else:
+        has_branched('parse_content_range', 4)
+        instance_length = None
+    
+    if content_range_dict['instance_length']:
+        has_branched('parse_content_range', 5)
+    else:
+        has_branched('parse_content_range', 6)
```
Comment (jberglinds) [inline httpie/downloads.py]: This used to check the statement above. Now it's duplicated so this should be removed.

**Comment by jberglinds** [comment]
I removed the last code duplication. Good to merge?

**Review by ghost — APPROVED** [review]
Looks good now, I think it can be merged

**Pull request merged**